vault/ufw.sls

```diff
@@ -0,0 +1,13 @@
+# configure ufw (firewall) policies for the nomad server and/or client
+
+{%- from "hashicorp/macro.sls" import render_app_ufw_formula with context %}
+
+# description for the service unit and ufw configuration files
+{%- set desc = salt['pillar.get']('vault:systemd_desc', "Hashiscorp Vault") %}
+{%- set server_desc = desc ~ ' Server' %}
```

Comment:
With vault, we only run the server, there is no client agent, just a cli client. I think it would be ok to include `Server` in the previous line, eg: `{%- set desc = salt['pillar.get']('vault:systemd_desc', "Hashiscorp Vault Server") %}`.